pynecone/components/base/link.py

```diff
@@ -19,3 +21,56 @@ def _render(self) -> Tag:
             href=self.href,
             rel=self.rel,
         )
+
+
+class ScriptTag(Component):
+    """A component that creates a script tag with the speacified type and source.
+
+
+    Args:
+        type: This attribute indicates the type of script represented.
```

Comment:
In a follow up, let's move these comments to be directly above each prop, like in the other component examples. This lets us automatically generate documentation for it on our main website.